bokehjs/src/lib/models/widgets/date_range_slider.ts

```diff
@@ -1,6 +1,5 @@
-import tz from "timezone"
-
 import {AbstractSlider, AbstractRangeSliderView} from "./abstract_slider"
+import tz from "timezone"
```

Comment:
The original was perfectly fine, to signify that we import an external dependency.